lib/rpc/server/peers.go

```diff
@@ -109,7 +109,7 @@ func (r *peers) tryPeer(ctx context.Context, peer *peer) error {
 }
 
 func (r *peers) monitorPeers() {
-	log := r.WithField("health.checker", r)
+	log := r.WithField("health.checker", r.String())
```

Comment:
This fixes `Failed to fire hook: json: unsupported type: func() string` in service logs.